cmd/events-main.go

```diff
@@ -53,7 +48,7 @@ COMMANDS:
 }
 
 // mainEvents is the handle for "mc events" command.
-func mainEvents(ctx *cli.Context) {
+func mainEvents(ctx *cli.Context) error {
```

Comment:
Basically errorIf never exited the program so we didn't have a consistent exit code.